public/app/features/dashboard/api/types.ts

```diff
@@ -14,8 +14,30 @@ export interface DashboardAPI<G, T> {
   deleteDashboard(uid: string, showSuccessAlert: boolean): Promise<DeleteDashboardResponse>;
 }
 
+export interface DashboardStatus {
+  /**
+   * When querying a dashboard that was stored in a different version than the current apiVersion,
+   * it will return conversion errors because conversions in the backend are not implemented yet
+   * @example If we query a dashboard that was stored in v2alpha1, from alhpaV1:
+   * {
+   *   "status": {
+   *     "conversion": {
+   *       "error": "conversion not implemented yet",
+   *       "failed": true,
+   *       "storedVersion": "v2alpha1"
+   *     }
+   *   }
+   * }
+   */
+  conversion?: {
+    error: string;
+    failed: boolean;
+    storedVersion: DashboardStoredVersion;
+  };
+}
+
```

Comment:
Can we use the auto-generated types from https://github.com/grafana/grafana/blob/main/packages/grafana-schema/src/schema/dashboard/v2alpha1/types.status.gen.ts#L1-L29 instead?

Reply:
Let me add it to the issue related to [SDK usage](https://github.com/grafana/grafana/issues/101956). I'd love that! Thanks for bringing this! 💯